src/pages/TruthTable.vue

```diff
@@ -183,8 +183,8 @@ export default defineComponent({
 
     const tableHeaders = shallowRef<MathJson[]>([]);
     const tableRows = ref<boolean[][]>([[]]);
-    const flipBits = ref<boolean>(false); // TODO: Make urlRef
-    const hideColumns = ref<boolean>(false); // TODO: Make urlRef
+    const flipBits = urlRef("flipped",false);
+    const hideColumns = urlRef("hideExtra",false);
```

Comment:
Could this be changed to `hide-extra`, just to follow the typical convention for URL query params?

Reply:
done, thx